test/Microsoft.Data.Analysis.Tests/ArrayComparer.cs

```diff
@@ -88,15 +88,19 @@ private void CompareArrays<T>(PrimitiveArray<T> actualArray)
             }
             else
             {
-                // expectedArray may have passed in a null bitmap. DataFrame might have populated it with Length set bits 
+                // expectedArray may have passed in a null bitmap. DataFrame might have populated it with Length set bits
                 Assert.Equal(0, expectedArray.NullCount);
                 Assert.Equal(0, actualArray.NullCount);
                 for (int i = 0; i < actualArray.Length; i++)
                 {
                     Assert.True(actualArray.IsValid(i));
                 }
             }
+#if NET
```

Comment:
What is this `#if` supposed to be doing? `NET_6_OR_GREATER`?

Reply:
It was meant to be for versions of .NET (as opposed to .NET Core or Framework) as I observed that unit tests for .NET Framework were failing with the enclosed assertion with an error about not being able to use `Span` any generic methods.

I think it's worth having the Equality assertion when possible as it makes it much faster debugging this test due to better error messages.

Reply:
Switched to the more widely used NET_6_OR_GREATER